acct-user/radius/radius-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="FreeRadius program user"
+ACCT_USER_ID=95
+ACCT_USER_HOME=/var/log/radius
```

Comment:
Home in /var/log? @orlitzky

Reply:
Yeah, thanks, this should be left at the default. The directory `/var/log/radius` is already created (with the right permissions) by the freeradius ebuild. Setting it as the home directory here, too, is going to cause problems down the line if the two ever get out-of-sync.

Reply:
I simply follow what it was been did by previous ebuilds. Why could cause problems? It's only the default home directory assigned to the created user.

Reply:
I've [explained the full rationale here](http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml), but basically there are three issues:

1. From a software engineering perspective, it's just in the wrong place. The user doesn't need `/var/log/radius` as its home directory, it's only the freeradius program that uses it. So, that directory should be handled by the freeradius ebuild.
2. Setting an unnecessary home directory inhibits the sharing of users, which was one of the main points of GLEP81. If somebody else wants to use the freeradius user, now they get `/var/log/radius` created on the system for no reason.
3. The GLEP81 eclass resets the ownership and permissions on `ACCT_USER_HOME` every time the package is (re-)emerged. This can lead to "surprises," and security vulnerabilities. If you or anyone else (like the end-user) tries to change the permissions on `/var/log/radius`, they'll get reset the next time the user package is installed. This is an unlikely vulnerability, but still one that we would rather not have if there's no good reason for it.

Reply:
ACCT_USER_HOME dropped.